examples/tensorflow/single-gpu-tensorflow.ipynb

```diff
@@ -0,0 +1,205 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ded426b2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def train_model_fit(n_epochs, base_lr, batchsize, classes):\n",
+    "\n",
+    "    model = tf.keras.applications.ResNet50(include_top=True, weights=None, classes=classes)\n",
+    "\n",
+    "    # Data\n",
+    "    train_ds = (\n",
+    "        tf.keras.preprocessing.image_dataset_from_directory(\n",
+    "            \"dataset/birds/train\", image_size=(224, 224), batch_size=batchsize\n",
+    "        )\n",
+    "        .prefetch(2)\n",
+    "        .cache()\n",
+    "        .shuffle(1000)\n",
+    "    )\n",
+    "    # printing sample birds images\n",
+    "    for birds, labels in train_ds.take(1):\n",
+    "        plt.figure(figsize=(18, 18))\n",
+    "        for i in range(9):\n",
+    "            plt.subplot(3, 3, i + 1)\n",
+    "            plt.imshow(birds[i].numpy().astype(\"uint8\"))\n",
+    "            plt.axis(\"off\")\n",
+    "    plt.show()\n",
+    "\n",
+    "    valid_ds = tf.keras.preprocessing.image_dataset_from_directory(\n",
+    "        \"dataset/birds/valid\", image_size=(224, 224), batch_size=batchsize\n",
+    "    ).prefetch(2)\n",
+    "\n",
+    "    optimizer = keras.optimizers.Adam(lr=base_lr)\n",
+    "    model.compile(loss=\"sparse_categorical_crossentropy\", optimizer=optimizer, metrics=[\"accuracy\"])\n",
+    "    start = time.time()\n",
+    "\n",
+    "    model.fit(\n",
+    "        train_ds,\n",
+    "        epochs=n_epochs,\n",
+    "        validation_data=valid_ds,\n",
+    "    )\n",
+    "    end = time.time() - start\n",
+    "    print(\"model training time\", end)\n",
+    "\n",
+    "    tf.keras.models.save_model(model, \"model/keras_single/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5896cdf-b8a6-416e-aeb2-db6d1cb49dd7",
+   "metadata": {},
+   "source": [
+    "In code below we are setting up necessary parameters. We are only running a few epochs to save time, but once you've got this working you'll have all the information you need to build and run bigger TensorFlow models on Saturn Cloud. A single GPU reviews all our batches every epoch. In this example's case, it sees 64 images per batch x 640 batches x 3 epochs. These are all done serially, nothing can be parallelized because only one processor is utilized. "
```

Comment:
```suggestion
    "In the code below, we set up the necessary parameters. We run only a few epochs to save time. But once you've got this model working, you'll have all the information you need to build and run bigger TensorFlow models on Saturn Cloud. A single GPU reviews all our batches every epoch. In this case, it sees 64 images per batch x 640 batches x 3 epochs. These batches are all computed serially; the calculations cannot be parallelized because only one processor is utilized. "
```